bedrock/privacy/templates/privacy/archive/firefox-cliqz-2018-06.de.html

```diff
@@ -0,0 +1,272 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "privacy/base-resp.html" %}
+
+{% block page_title %}Mozilla Firefox + Cliqz-Datenschutzhinweis — Archiviert am 26. Juni 2018{% endblock %}
```

Comment:
Note: currently this locale specific template won't load when `Dev=False` due to [bug 1415964](https://bugzilla.mozilla.org/show_bug.cgi?id=1415964). Until that it fixed, I'm not sure of the best solution other than adding an empty lang file to activate 'de' in the l10n repo. This isn't ideal though :/

Any other ideas @pmac @jpetto?

Reply:
Best option I think is to add an activated lang file to the template. If you specifically add a lang file to a template it also inherits all of its tags.

Reply:
> Best option I think is to add an activated lang file to the template. If you specifically add a lang file to a template it also inherits all of its tags.

Is the danger with doing this that it may also activate the locale for other languages too? We just need `en-US` and `de` in this case.

Reply:
Yes. It would. Not sure we can work around this system at the moment. Might just have to submit a PR for the l10n repo to add the empty file with the active tag.

Reply:
Submitted PR https://github.com/mozilla-l10n/www.mozilla.org/pull/236

Reply:
Ok, this has been fixed via the above PR. I've tested locally with `Dev=False` and things are working.